PARTNERS.md

```
@@ -13,20 +13,27 @@ better stability and maintainability.
-| 4.16              | 1.4          | v1.14          | released                      | 
-| 4.17              | 1.4          | v1.14          | released                      |
-| 4.18              | 1.4          | v1.14          | released                      |
-| 4.19              | 1.5          | v1.16          | released                      |
-| 4.20              | 1.5          | v1.16          | released                      |
-| 4.21              | 1.5          | v1.16          | Q1 2026                       |
-| 4.22              | 1.6          | v1.18          | Q3 2026                       |
-| 4.23              | 1.6          | v1.18          | Q1 2027                       |
-| 5.0               | 1.6          | v1.18          | Q3 2026                       |
-| 5.1               | 1.7          | v1.20          | Q2 2027                       |
-| 5.2               | 1.7          | v1.20          | Q3 2027                       |
+| OpenShift Version | OADP Version | Velero Version | Estimated Release Timeline [1]| Extended lifecycle support |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)